src/plugins/intel_gna/frontend/quantization.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+#include <vector>
+#include <cstdint>
+#include "backend/gna_types.h"
+
+#define MAX_OUT_MULTIPLIER 230
+#define MAX_VAL_1B_WEIGHT 127
+#define MAX_VAL_1B_FEAT 64
+#define MAX_VAL_1B_BIAS 127
+#define MAX_VAL_2B_WEIGHT 16384
+#define MAX_VAL_2B_FEAT 16384
+#define MAX_VAL_4B_BIAS 1073741824
+
+struct QuantizationCallbackData {
+    const size_t num_rows;
+    const size_t num_columns;
```

Comment:
Why do quantization should distinguish rows and columns?
Why `size` is not enough?
eg. Is quantization also responsible for some padding?

Reply:
That was not a part of my considerations for this PR, but in the old code there was some dead code for padding. I'll have to verify if this can be removed safely.

Reply:
Not possible due to compound bias.